backend/src/test/java/com/cruru/util/fixture/QuestionFixture.java

```diff
@@ -23,11 +23,11 @@ public static Question createDropdownQuestion(ApplyForm applyForm) {
         return new Question(DROPDOWN, "객관식 단일 선택", "단 하나만 골라주세요.", 3, false, applyForm);
     }
 
-    public static Question createCheckboxQuestion(ApplyForm applyForm) {
-        return new Question(CHECK_BOX, "체크 선택", "체크해주세요.", 4, false, applyForm);
+    public static Question createMultipleChoiceQuestion(ApplyForm applyForm) {
+        return new Question(MULTIPLE_CHOICE, "객관식 다중 선택", "모두 선택해주세요..", 4, false, applyForm);
```

Comment:
```suggestion
        return new Question(MULTIPLE_CHOICE, "객관식 다중 선택", "모두 선택해주세요.", 4, false, applyForm);
```
여기에 점이 두개인거 빼고는 완벽하네요.

Reply:
수정했습니다 :)